ddtrace/propagation/http.py

```diff
@@ -627,8 +628,13 @@ def _get_tracestate_values(ts):
                 sampling_priority_ts_int = None
 
             origin = dd.get("o")
```

Comment:
Can we replace `~` to `=` once?
```suggestion
            # we encode "=" as "~" in tracestate so need to decode here
            dd = dd.replace("~", "=")
            origin = dd.get("o")
```

Reply:
I don't think `replace` is a dict command, but we could do a dict comprehension if you think that'd be better? However then we'd be doing two dictionary comprehensions. One for the the char replacement, and one for creating `other_propagated_tags` below.